requirements.txt

```diff
@@ -9,3 +9,4 @@ PyQt5>=5.7.1
 control>=0.7.0
 Sphinx>=1.4.9
 sphinx-rtd-theme>=0.1.9
+pybind11>=2.6.0b1
```

Comment:
Remember to bump once the release is out

Reply:
yes, it is only for testing now and not the final version.